.github/workflows/benchmarks.yml

```diff
@@ -30,6 +30,7 @@ jobs:
           -DCMAKE_C_COMPILER=${{matrix.compiler.c}}
           -DCMAKE_CXX_COMPILER=${{matrix.compiler.cxx}}
           -DUMF_BUILD_BENCHMARKS=ON
+          -DUMF_BUILD_BENCHMARKS_MT=ON
```

Comment:
You could use TSAN in these tests.

Reply:
I'm not sure, those are not really tests and we should already be testing those paths in other places.